library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/Count.java

```diff
@@ -26,7 +26,7 @@
  *
  * @since 3.5.0
  */
-public class Count implements Accumulation<Long, MutableLong, Long>
+public class Count<T> implements Accumulation<T, MutableLong, Long>
```

Comment:
It doesn't make sense to parameterize Count class as the type of input really doesn't matter.